nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs

```diff
@@ -343,6 +389,10 @@ internal static async Task<ImmutableArray<ProjectBuildFile>> LoadBuildFiles(stri
             });
             buildFileList.AddRange(project.Imports.Select(i => i.ImportedProject.FullPath.NormalizePathToUnix()));
         }
+        catch(InvalidProjectFileException)
```

Comment:
nit:
```suggestion
        catch (InvalidProjectFileException)
```

Reply:
ignore if the current formatting satisfies the .editorconfig

Reply:
Cleanup on save got turned off somehow, turned it back on, thanks!